packages/transaction-controller/CHANGELOG.md

```diff
@@ -7,6 +7,36 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [37.0.0]
+
+### Changed
+
+- Remove unapproved transactions during initialisation ([#4658](https://github.com/MetaMask/core/pull/4658))
+- Fail approved and signed transactions during initialisation ([#4658](https://github.com/MetaMask/core/pull/4658))
+
+### Removed
+
+- **BREAKING:** Remove `initApprovals` method ([#4658](https://github.com/MetaMask/core/pull/4658))
+- **BREAKING:** Remove `beforeApproveOnInit` hook ([#4658](https://github.com/MetaMask/core/pull/4658))
+- **BREAKING:** Remove `TraceContext`, `TraceRequest`, and `TraceCallback` types ([#4655](https://github.com/MetaMask/core/pull/4655))
+  - These were moved to `@metamask/controller-utils`.
```

Comment:
```suggestion
- Remove `TraceContext`, `TraceRequest`, and `TraceCallback` types ([#4655](https://github.com/MetaMask/core/pull/4655))
   - These were moved to `@metamask/controller-utils`.

### Removed

- **BREAKING:** Remove `initApprovals` method ([#4658](https://github.com/MetaMask/core/pull/4658))
- **BREAKING:** Remove `beforeApproveOnInit` hook ([#4658](https://github.com/MetaMask/core/pull/4658))
```

This type changes are not breaking because they used internally, not exported from package before.